File: java/sinmple_example (1).java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.net.HttpURLConnection;
import java.net.URL;
import java.net.URLEncoder;
import java.nio.charset.StandardCharsets;
import java.util.HashMap;
import java.util.Map;
import java.util.stream.Collectors;

public class Main {
    public static void main(String[] args) {
        try {
            String webhookId = "YOUR_WEBHOOK_ID";
            String webhookCode = "YOUR_WEBHOOK_CODE";
            String urlStr = "https://psyll.com/env/webhook/" + webhookId;

            Map<String, String> payload = new HashMap<>();
            payload.put("code", webhookCode);
            payload.put("action", "BUY");
            payload.put("quantity", "0.0034");
            payload.put("symbol", "BTCUSDT");

            String formData = payload.entrySet().stream()
                .map(e -> {
                    try {
                        return URLEncoder.encode(e.getKey(), StandardCharsets.UTF_8.toString()) + "=" +
                               URLEncoder.encode(e.getValue(), StandardCharsets.UTF_8.toString());
                    } catch (Exception ex) {
                        return "";
                    }
                })
                .collect(Collectors.joining("&"));

            URL url = new URL(urlStr);
            HttpURLConnection conn = (HttpURLConnection) url.openConnection();
            conn.setRequestMethod("POST");
            conn.setRequestProperty("Content-Type", "application/x-www-form-urlencoded");
            conn.setDoOutput(true);

            try (var outputStream = conn.getOutputStream()) {
                outputStream.write(formData.getBytes(StandardCharsets.UTF_8));
            }

            int statusCode = conn.getResponseCode();
            System.out.println("Status code: " + statusCode);

            StringBuilder response = new StringBuilder();
            try (BufferedReader reader = new BufferedReader(new InputStreamReader(conn.getInputStream()))) {
                String line;
                while ((line = reader.readLine()) != null) {
                    response.append(line);
                }
            }

            System.out.println("Response:");
            System.out.println(response.toString());

            conn.disconnect();
        } catch (Exception e) {
            System.err.println("Error: " + e.getMessage());
        }
    }
}

File: kotlin/sinmple_example (1).kt
import okhttp3.FormBody
import okhttp3.OkHttpClient
import okhttp3.Request
import java.io.IOException

fun main() {
    val webhookId = "YOUR_WEBHOOK_ID"
    val webhookCode = "YOUR_WEBHOOK_CODE"
    val url = "https://psyll.com/env/webhook/$webhookId"

    val client = OkHttpClient()
    val formBody = FormBody.Builder()
        .add("code", webhookCode)
        .add("action", "BUY")
        .add("quantity", "0.0034")
        .add("symbol", "BTCUSDT")
        .build()

    val request = Request.Builder()
        .url(url)
        .post(formBody)
        .build()

    try {
        client.newCall(request).execute().use { response ->
            println("Status code: ${response.code}")
            val responseBody = response.body?.string()
            println("Response:")
            println(responseBody)
        }
    } catch (e: IOException) {
        println("Error: ${e.message}")
    }
}